dfdaemon/seed/manager.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package seed
+
+import "time"
+
+const (
+	//  defaultDownloadConcurrency is the default concurrency limit of prefetching seed.
+	defaultDownloadConcurrency = 4
```

Comment:
defaultDownloadConcurrency is unused at this moment, may be removed now?

Reply:
fix it.